statsmodels/tools/validation/validation.py

```diff
@@ -395,6 +371,27 @@ def float_like(value, name, optional=False, strict=False):
 
 
 def string_like(value, name, optional=False, options=None, lower=True):
+    """
+    Check if of type string and raise not string_like if not
+
+    Parameters
+    ----------
+    value : object
+        Value to verify
+    name : str
+        Variable name for exceptions
+    optional : bool
+        Flag indicating whether None is allowed
+    options : string
```

Comment:
Type should be `Sequence[str]` or `tuple[str]`

Reply:
@bashtage I did what you suggested above
- restored blank lines between code sections
- re-phrased the docstring
- replace str with Sequence[str] or tuple[str]
Thank you very much